app/forms/api/v1/appointment.rb

```diff
@@ -32,7 +32,7 @@ def create
       private
 
       def end_at
-        Time.zone.parse(start_at) + 1.hour
+        Time.zone.parse(start_at) + 70.minutes
```

Comment:
Actually we don't even need the 70 minutes if we refactor how the query works. I opted for this route because it was faster, though.

I'm feeling like that was maybe the wrong choice...